### ISSUE TITLE
Can u add the reconnect Method for NettyWebSocket.

### ISSUE THREAD

**Issue by zhp0260** [AsyncHttpClient/async-http-client#1709]
last year, i used the websocket function, I find when the server disconnect the nettywebsocket client, then I don't know how reconnect the server side.  I just only close this asyncHttpClient and renew a websocket. I think it is not well for it, the framework should be support reconnect, make it for reuse.

**Comment by slandelle** [comment]
Contributions welcome.

**Comment by sivaalli** [comment]
I can take a look at it.

**Comment by slandelle** [comment]
@sivaalli Sure, feel free to take a stab at it. Thanks.

**Comment by salliEG** [comment]
Agreed. I do not specifically need this use case. If there is any traction on this request I will revisit this. :)

**Comment by sivaalli** [comment]
Agreed. I do not specifically need this use case. If there is traction on this request, I will revisit. Thanks :)

**Comment by TomGranot** [comment]
@slandelle I propose we close this since indeed no traction.

**Comment by pcg76** [comment]
> Agreed. I do not specifically need this use case. If there is traction on this request, I will revisit. Thanks :)

Hi, I need this reconnect feature for camel route. May I know where to look at and hopefully resolve this issue. Thanks!
